vit/readline.py

```diff
@@ -55,7 +55,7 @@ def keypress(self, key):
                     + self.edit_obj.get_edit_text()[self.edit_obj.edit_pos + 1 :])
             return None
         # Delete next character.
-        elif key in ('delete', 'ctrl d'):
+        elif key in ('ctrl d'):
```

Comment:
We need to have a comma after ```'ctrl d'```, otherwise I don't think Python will see that as a single item tuple. See the other conditionals checks in that section.

Reply:
Yeah, sorry about that. Should have been testing these past changes.

Though now while testing I remembered why I added `delete` to readline like that. It seems that its implementation by urwid is kinda buggy, at least for me. Whenever I press `delete`, it deletes the next character but also positions the cursor at the end of the command line.

This also happens on the 2.x branch, so could you please also test it on your end?

Reply:
I'm not seeing that behavior in my environment -- ```delete``` is functioning as expected, both deleting from the end of a line and from the middle of a line.

I would be pretty surprised if the Urwid library had such a bug -- properly deleting text in the middle of a string should be supported in the basic edit box functionality.

Can you try to reproduce the issue on your end? Maybe there's something specific you're doing that I didn't try.

Reply:
I just tested a sample code using just urwid and it worked fine so it really isn't a bug in urwid.

Though I also tested using a clean vit from pip and a clean taskwarrior and vit config, and the issue does happen. So it doesn't have to do with the patch, nor with my config. But if you can't reproduce it then it should be something to do with my environment. Will have to test more.

The issue isn't anything specific. Everytime I press `delete` in any command bar inside vit, be it the prompt for typing a command or the `Add:` prompt, it deletes the next character but also positions the cursor at the end of the line. When I overrode the handling of `delete` with my readline module (as in this code snippet), the issue didn't happen.

Reply:
How do you want to proceed?
1. Continue to investigate the bug in your local environment to see if you can isolate why it's happening, or...
2. Complete the PR without re-adding ```delete``` to the Readline shortcuts at this time

Reply:
Okay, since this issue seems to be specific to my environment, it makes more sense to proceed this PR removing `delete` (option 2) and later I can investigate this problem further and open an issue for it.